login-utils/newgrp.c

```diff
@@ -187,7 +187,7 @@ int main(int argc, char *argv[])
 {
 	struct passwd *pw_entry;
 	struct group *gr_entry;
-	char *shell;
+	char *shell, *command = NULL;
 	int ch;
 	static const struct option longopts[] = {
 		{"version", no_argument, NULL, 'V'},
```

Comment:
You need to add "command" (long option) too.

Reply:
Done.